java-library/build.gradle

```diff
@@ -28,6 +28,7 @@ sourceCompatibility = 1.7
 targetCompatibility = 1.7
 
 dependencies {
+    compile project(':kmp-library')
```

Comment:
Until everything is ported, the legacy java-lib depends on the multiplatform library so we can reuse the existing tests for our changes and make sure we don't break the api.